src/docfx/build/metadata/MetadataProvider.cs

```diff
@@ -91,7 +88,10 @@ public MetadataProvider(Docset docset, Cache cache)
 
             errors.AddRange(_schemaValidator.Validate(result));
 
-            return (errors, metadata);
+            var (validationErrors, metadataModel) = JsonUtility.ToObject<InputMetadata>(result);
+            errors.AddRange(validationErrors);
+
+            return (errors, result, metadataModel);
```

Comment:
so there is some content overlapping between `result` and `metadataModel`?

Reply:
result will be only for reading